### DISCUSSION TITLE
Use GitHub Sponsors?

### DISCUSSION THREAD

**Discussion by gruenich** [material-table-core/core#274] [General]
When I click on "Sponsor", I am redirected to OpenCollective. I think it is a good idea to use GitHub sponsors, as this might grow to a larger sponsoring base for donation form companies. What do you think?

In general, what would you do with donations / sponsorships? Could I buy features, bug fixes or general improvements, whatever you think would be best for the project?

**Comment by matthewoestreich** [comment]
When you say buy features do you mean something similar to what material UI does? Personally, I would like to have all the same features but for free. We try to fix bugs/add features/etc regardless of donations. If you have an issue that needs to be addressed, you do not have to donate for us to resolve said issue. I hope that helps!

**Reply by wgehring** [reply]
Following up on gruenich's question (we're at the same company): You have a "Sponsors" button, so we assume you might accept a donation as an appreciation for your great work? (And at the moment, we can make sponsorship donations only through GitHub sponsors, not yet Open Collective, thus the first part of the question.)

**Reply by matthewoestreich** [reply]
Ok cool that makes sense. I'm sure we can add that (I did not set up the donations so I would want to run it by @Domino987 first).

Thank you for taking interest in this project. It means a lot :)

**Reply by wgehring** [reply]
Ok great! Thank _you_. Drop us a note here if you can set it up. 
(We actually need just a little bit more time on our end, too, to make some final steps for our sponsoring stuff, but we wanted to reach out to you to see if you might be interested.)

**Reply by matthewoestreich** [reply]
No worries at all. Your generosity is greatly appreciated! I look forward to working together. :)

**Reply by Domino987** [reply]
Hi thank you for the interest. I will look into it through github.

Currently, we have no exact setup of what to do with the donations and we will keep everything (features/bug fixes) available for all!!
I will spend more time on this repo if I can allow to allocate more time from my other responsibilities through donations towards it.

Optional, we could work on a system for custom support on certain problems/prioritized bug fixes.

But this will be decided when requests and requirements from donors would come in, I guess.
We are open for suggestions as well.
Great to see more interesting coming towards this repo.
I hope we will be able to make this a great library for all.

**Reply by wgehring** [reply]
You mean, "an even greater library". ;-) 
Thanks - and let us know when / if you can set this up through GitHub. (They are happy for every one who participates as they are interested to making this work, too, of course.)

**Reply by wgehring** [reply]
Hello @Domino987 ,
any news on sponsorship through GitHub sponsors? :-)
We still have you on our list and would make another attempt to sponsor you. :-)

I spoke to GitHub; they looked at material-core-table and said you would need to update your host information or payment details.

**Reply by Domino987** [reply]
So yes, apparently there was a problem with the fiscal host that I was not told about. But it got resolved and I expect it to be live by tomorrow.

**Reply by wgehring** [reply]
Ah nice! Sounds good. 
Let me know once it's finalized. :-)

**Reply by Domino987** [reply]
https://github.com/sponsors/material-table-core?o=esb
It finally is!

**Reply by wgehring** [reply]
Ok cool! Thank you.
Our plan is to finalize this round of sponsorhips sometime in February. You will hear form us again then (and receive a donation). :-)

**Reply by Domino987** [reply]
Any news on this 😁

**Reply by wgehring** [reply]
Hi @Domino987 and sorry for my late reply. Sponsorship is coming! Also sorry for the delay which is basically my fault. I just ask for a liiitle bit more patience.  :-)
